chain/processor.go

```diff
@@ -167,28 +167,28 @@ func (p *Processor) Execute(
 	}
 
 	if isNormalOp {
-		if err := p.validityWindow.VerifyExpiryReplayProtection(ctx, b, parentTimestamp); err != nil {
-			return nil, err
+		oldestAllowed := max(0, b.Tmstmp-r.GetValidityWindow())
```

Comment:
should the validity window object itself be responsible for calculating the "oldest allowed" ? It seems that we had a bug due to improper code duplication.